trigger/streamtester/timer.go

```diff
@@ -256,7 +257,10 @@ func ReadCsvInterval(path string, emitInfo *HandlerEmitterInfo) ([]string, error
 			return nil, err
 		}
 		emitInfo.Lines = data
-
+		defer func() {
```

Comment:
Let me under stand this, you are returning the header row, but increasing the count by 1.  What is count?  Maybe you should rename count to something more informative.  maybe "Index" or "RowNum" or something if thats what it is.

Reply:
Changed it to CurrentIndex for better clear understanding.